src/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Wbpack Exercise</title>
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet">
+    </head>
+    <body>
+        <section>
+            <ul id="complete-list">
+                <li class="top-list-item">
+                    <div class="flex">
+                        <span>Today's Todo</span>
+                        <span class="material-icons">cached</span>
+                    </div>
+                    <hr>
+                </li>
+
+                <li class="top-list-item">
+                    <div class="flex">
+                        <input id="input" placeholder="Add to your list"
+                            type="text">
+                        <span class="material-icons">
+                            keyboard_return
+                        </span>
+                    </div>
+                    <hr>
+                </li>
+
+                <li id="button-li"><button type="button">Clear All Completed</button></li>
```

Comment:
What do you think about taking this button outside the ul tag since it is not a todo task

Reply:
Do you mean if you tick it will show a green tick and then you can't tick Peter? if that is the case that is by design not by mistake. should I change it?

Reply:
OOh really, I thought that was a mistake